docs/DefineDifferentiability.md

```diff
@@ -0,0 +1,99 @@
+# A Short Guide on Defining Differentiability for ONNX operators
+
+The differentiability definition of an operator consists of several aspects.
+
+- Differentiable inputs, which can be referenced in Gradient's `xs` attribute.
+- Differentiable outputs, which can be referenced in Gradient's `y` attribute.
+- The math equation to compute the Jacobian matrix (or tensor). If a variable (input or output) is differentiable or not is judged by math. If the Jacobian matrix (or tensor) exists, then the considered operator has some differentiable inputs and outputs.
+
+There are several strategies to implement auto-differentiation such as forward accumulation, backward accumulation, and dual variable.
+Because most deep learning frameworks are backward-based, the reviewers should ensure the PR authors provide enough details on that.
```

Comment:
I didn't quite understand. I guess this means that the PR should contain details on how the backward-prop can be implemented? The subsequent discussion seems to imply that this is not necessary? Is this optional?

Reply:
No. The author here means "author of differentiability tag", not the author of operator.